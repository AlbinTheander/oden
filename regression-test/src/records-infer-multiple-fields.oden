package main

a = {
  p1 = "p1",
  p2 = "p2",
  p3 = {
    p4 = "p4",
    p5 = "p5"
  }
}


describe : { p1: string, p2: string, p3: { p4: string, p5: string } } -> ()
describe(thing) = {
  println(thing.p1)
  println(thing.p2)
  println(thing.p3.p4)
  println(thing.p3.p5)
}

main() = describe(a)
