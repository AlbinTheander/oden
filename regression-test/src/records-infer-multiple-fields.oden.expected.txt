p1
p2
p4
p5
